nodebuilder/p2p/network_test.go

```diff
@@ -0,0 +1,26 @@
+package p2p
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestChainIDFromNetwork(t *testing.T) {
+	var tests = []struct {
+		name            string
+		expectedChainID string
+	}{
+		{name: "mocha-3-da-custom", expectedChainID: "mocha-3"},
```

Comment:
How are these tests still passing if : is the separator now?

Reply:
I forgot to update tests, my bad. I was pushing code while on call with john lol.